Example/Custom Integration/CardManualIntegrationExampleViewController.m

```diff
@@ -0,0 +1,143 @@
+//
+//  CardManualIntegrationExampleViewController.m
+//  Custom Integration (ObjC)
+//
+//  Created by Ben Guo on 2/22/17.
+//  Copyright © 2017 Stripe. All rights reserved.
+//
+
+#import <Stripe/Stripe.h>
+#import "CardManualIntegrationExampleViewController.h"
+#import "BrowseExamplesViewController.h"
+
+/**
+ This example demonstrates creating a payment with a credit/debit card using Manual Integration.
+ It creates a Payment Method using card information collected with STPPaymentCardTextField, and
+ then sends the Payment Method ID to our example backend to create and confirm the Payment Intent.
+ */
+@interface CardManualIntegrationExampleViewController () <STPPaymentCardTextFieldDelegate, UIScrollViewDelegate>
```

Comment:
Let's call this `CardManualConfirmationExampleViewController` instead, in line with the terminology here https://stripe.com/docs/payments/payment-intents/quickstart#flow-manual-confirmation

Reply:
👍